README.md

```diff
@@ -21,26 +21,6 @@ This is the 'base' layer for any transport that wants to be used by libp2p and i
 > gx-go rewrite
 ```
 
-## Usage
```

Comment:
no new usage text? @Stebalien tsk tsk tsk

Reply:
One generally doesn't *use* transports directly like this. One implements them.

... fine. I'll add an example.